js/index.js

```diff
@@ -63,6 +63,37 @@ function displayProjects() {
   projectCards.appendChild(cards);
 }
 
+const handleSubmit = (event) => {
+  event.preventDefault();
+  const nameError = document.getElementById('name-error');
+  const emailError = document.getElementById('email-error');
+  const msgError = document.getElementById('msg-error');
+  const name = document.getElementById('name-input').value;
+  const email = document.getElementById('email-input').value;
+  const msg = document.getElementById('msg-input').value;
+  let error = false;
+  if (name.length === 0) {
+    nameError.innerHTML = 'Name is required';
+    error = true;
+  }
+  if (email.length === 0) {
+    emailError.innerHTML = 'Email is required';
+    error = true;
+  }
+  if (!email.match('[a-z0-9]+@[a-z]+.[a-z]{2,3}')) {
+    emailError.innerHTML = 'Email is incorect';
+    error = true;
+  }
+  if (msg.length === 0) {
+    msgError.innerHTML = 'Message is required';
+    error = true;
+  }
+  if (!error) {
+    const form = document.getElementById('contact-form');
+    form.submit();
+  }
+};
+
```

Comment:
**[REQUIRED CHANGES]**

- Hey, great job with the validation, however, your form is still submitted when the email is in uppercase. Please, kindly fix this issue and ensure you add the error message either just above the submit button or beneath the submit button.

Reply:
I have fixed it